openmdao/docs/features/core_features/working_with_derivatives/checking_partials.rst

```diff
@@ -49,10 +49,10 @@ are fairly large, so we give it a higher stepsize.
 .. embed-test::
     openmdao.core.tests.test_check_derivs.TestCheckPartialsFeature.test_set_step_on_comp
 
-Here, we show how to set the method. In this case, we use Complex Step on TrickyParaboloid because the finite difference is
+Here, we show how to set the method. In this case, we use complex step on TrickyParaboloid because the finite difference is
 less accurate.
 
-**Note**: You need to set `force_alloc_complex` to True during setup to utilize Complex Step during a check.
+**Note**: You need to set `force_alloc_complex` to True during setup to utilize complex Step during a check.
```

Comment:
minor: don't think Step needs to stay capped here...but don't let it hold you up, I can fix it on my current doc review branch